components/accelerator/nvidia/query/nvml/gsp_firmware.go

```diff
@@ -22,8 +22,15 @@ func GetGSPFirmwareMode(uuid string, dev device.Device) (GSPFirmwareMode, error)
 	}
 
 	gspEnabled, supported, ret := dev.GetGspFirmwareMode()
+	if IsNotSupportError(ret) {
+		mode.Enabled = false
+		mode.Supported = false
+		return mode, nil
```

Comment:
@cardyok This is the fix for

![image (1)](https://github.com/user-attachments/assets/8c3722c1-ea0b-42de-89a4-35655445099e)